server/src/main/java/org/opensearch/indices/cluster/IndicesClusterStateService.java

```diff
@@ -170,7 +173,8 @@ public IndicesClusterStateService(
         final GlobalCheckpointSyncAction globalCheckpointSyncAction,
         final RetentionLeaseSyncer retentionLeaseSyncer,
         final SegmentReplicationCheckpointPublisher checkpointPublisher,
-        final RemoteStorePressureService remoteStorePressureService
+        final RemoteStorePressureService remoteStorePressureService,
```

Comment:
Do we need to inject `RemoteStorePressureService` as well?

Reply:
Updated. Resolving this thread. Please feel free to re-open if needed.